src/xcode/ENA/ENA/Source/Services/TicketValidation/Models/ValidationDecorator/DecoratorServiceIdentityDocumentError.swift

```diff
@@ -0,0 +1,17 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+enum DecoratorServiceIdentityDocumentError: Error {
+ case VD_ID_CLIENT_ERR
+ case VD_ID_NO_NETWORK
+ case VD_ID_NO_ATS
+ case VD_ID_NO_ATS_SIGN_KEY
+ case VD_ID_NO_ATS_SVC_KEY
+ case VD_ID_NO_VS
+ case VD_ID_NO_VS_SVC_KEY
+ case VD_ID_SERVER_ERR
+ case VD_ID_PARSE_ERR
+ case VD_ID_EMPTY_X5C
```

Comment:
We split up the errors to errors associated with HTTP and errors associated with the validation itself so that the request can "capsulated" his errors.
So my proposal would be 2 enums:

The one for the validation stays here and looks like this:
```
enum ServiceIdentityDocumentValidationDecoratorError: Error {
case VD_ID_NO_ATS
case VD_ID_NO_ATS_SIGN_KEY
case VD_ID_NO_ATS_SVC_KEY
case VD_ID_NO_VS
case VD_ID_NO_VS_SVC_KEY
case VD_ID_EMPTY_X5C
}
```

and the other would be defined in your `DecoratorIdentityDocumentResource ` and looks like this:
```
enum ServiceIdentityDocumentValidationDecoratorResourceError: Error {
case VD_ID_CLIENT_ERR
case VD_ID_NO_NETWORK
case VD_ID_CLIENT_ERR
case VD_ID_NO_NETWORK
}
```